src/share/classes/com/sun/javatest/regtest/exec/RegressionScript.java

```diff
@@ -1139,6 +1140,10 @@ Map<String, String> getTestProperties()  {
         if (testQuery != null) {
             p.put("test.query", testQuery);
         }
+        Verbose verbose = params.getVerbose();
+        if (verbose != null) {
+            p.put("test.verbose", verbose.toString());
+        }
```

Comment:
You should not need the guard if you implemented `load/`save` in `RegressionParamaters`

Reply:
Same goes for the `testQuery` field above, right?

Reply:
I'll keep both guards in place.